src/index.ts

```diff
@@ -29,7 +29,7 @@ const ensureUrlEndsWithSlash = (url: string) => {
  * then Danger will comment with a warning on the pull request asking the developer
  * to include the JIRA issue identifier in the pull request title.
  */
-export default function jiraIssue(options: Options) {
+export default function jiraIssue(options?: Options) {
```

Comment:
If you're going to raise, I think you have the best description before - in this case the test - `expect(() => jiraIssue()).toThrow()` needs to change. Perhaps to something like `expect(() => (jiraIssue as any)()).toThrow()`